portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,50 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Angello's Portfolio</title>
+    <!-- My stylesheets -->
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="container">
+      <div class="nav-wrapper">
+        <div class="nav-left">
+          <div class="nav-link">
+            <a href="index.html">Home</a>
+          </div>
+          <div class="nav-link">
+            <a href="about.html">About me</a>
+          </div>
+          <div class="nav-link">
+            <a href="projects.html">Projects</a>
+          </div>
+        </div>
+
+        <div class="nav-right">
+          <div id="nav-name">
+            Angello Parrolivelli
+          </div>
+        </div>
+      </div>
+
+      <div id="content">
+        <div class="header">
+          <h1 class="g-green">Hi there!</h1>
+          <div id="intro">My name's <span class="g-blue">Angello</span>, and this is the portfolio I made in Google's SPS program!</div>
+        </div>
+
+        <hr>
+        
+        <div class="options">
+          <h2>You can read more about me <a class="g-yellow" href="about.html">here</a></h2>
```

Comment:
Semantically these are not headers that outline structure of your document, but rather paragraphs with a particular style (you probably wanted the text to be bigger and bolder). I'd probably use <p class="">